space_age/spec/features/user_can_view_index_page_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+feature "user can view index page" do
+  scenario "user will see full options for images" do
+    destination = create_list(:destination, 3)
+    visit root_path
```

Comment:
So I wrote a test for the landing page, the trip index and the show page. Destinations was just what I used for trip but we can switch the name. Same idea. Your breakdown of trips and destinations makes sense.